packages/Liquid-Tests.package/LQChoicesQuestionTests.class/instance/setUp.st

```diff
@@ -0,0 +1,4 @@
+running
+setUp
+	super setUp.
+	question := LiquidChoicesQuestion new.
```

Comment:
[sand] Creating the question-to-test in setup makes the tests somewhat hard to read, as you have to jump around the class and do some mental work to figure out what question is actually being tested. AFAICT, local variables inside of the tests would work just fine. What am I missing?

Reply:
Since we need a `LiquidChoicesQuestion` in all tests of this class, I decided to move the initialization of this variable to setUp. We could of course get the same behaviour by using a temporary variable, but I think it is more "idiomatic" (or whatever this falls under) to use setUp in this case - we could however choose a better fitting name for the variable, if you like

Reply:
I also think setting up state in setUp is pretty standard behaviour, and I don't think it really matters in a case that is this simple

Reply:
if there are reasons for `setUp` other than being "idiomatic", it's fine. we shouldn't follow idioms if they don't improve things, though. having the setUp logic made the test harder to read for me. temporary variables would've allowed to read the test without hopping between functions.

Reply:
I understand your perspective. However, I think that _writing_ tests will be a lot easier when using setUp, since we can skip the whole initialization of "stuff", just as we did e.g. in the ObjectRepoTests when it comes to starting the Server. Of course, in this case it is just assigning one variable, but I still believe it is worth it - especially since you only need to check once what `self question` is, but save yourself two lines in every future test

Reply:
I added an abstract superclass in the newest commit, please both @PaulVII and @Skn0tt take a look if you like it, else we can remove it again to work on it further

Reply:
what is "the latest commit"? 😅

Reply:
btw: you *can* delete comments and recreate them in the right conversation

Reply:
The one saying "Added an abstract superclass" ^^

Reply:
> btw: you _can_ delete comments and recreate them in the right conversation

Of course, but I didn't want to just delete it here, in case someone already saw it, as I noticed quite late, so I just added the note and copied the comment over to the correct conversation

Reply:
d3c61e6 you mean?